### PR TITLE
Correct last example

### PR DESCRIPTION
**Pull request by mrlife**
The last example was incorrect. I updated with intention to achieve the same result of 4. This differentiates the last example from the preceding one.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: samples/snippets/csharp/language-reference/builtin-types/EnumType.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static void Main()
                 var b = (Season)1;
                 Console.WriteLine(b);  // output: Summer
 
-                var c = (Season)4;
+                var c = (int)Season.Winter;
```
Comment (pkulikov) [inline samples/snippets/csharp/language-reference/builtin-types/EnumType.cs]: That would make `c` equal 3, not 4 as intended. Intention here is to show that one can convert *any* integer to the enum type.

Comment (mrlife) [reply samples/snippets/csharp/language-reference/builtin-types/EnumType.cs]: Sorry about that, meant to change result to 3.

Comment (pkulikov) [reply samples/snippets/csharp/language-reference/builtin-types/EnumType.cs]: Then it becomes the example of converting enum to int. That's already covered by the first example. I don't think any changes are required here.

**Comment by mrlife** [comment]
@pkulikov 

> Intention here is to show that one can convert any integer to the enum type.

My mistake. I was not sure the purpose of the example, so I learned something.

**Pull request closed without merging**